resources/assets/css/_components.css

```diff
@@ -159,15 +159,15 @@
 }
 
 .pagination-pages-mobile {
-    @apply flex justify-center px-2 mb-3 rounded bg-theme-primary-100;
+    @apply flex justify-center px-2 mb-3 rounded-xl bg-theme-primary-100;
```

Comment:
this is technically a button, so should be `rounded`. it affects the top button on mobile
<img width="325" alt="image" src="https://user-images.githubusercontent.com/35610748/115021570-6b874700-9ebc-11eb-8fcc-51cdb5936d2b.png">